galaxy_skeleton2.R

```diff
@@ -4,6 +4,8 @@
 
 # More detailed steps to complete Problem 1.
 
+setwd("/Users/michaeljosvawoldikechristensen/Desktop/Master_NHH/R_programming_for_Data_Science/Git_R_programmering/mj_assignment1/git-MichaelJosva")
```

Comment:
You dont need to add this line, as this will be different for any other people reading your script.